### PR TITLE
Rework tournament country rankings article

### PR DESCRIPTION
**Pull request by Joehuu**
Probably need a official check for this, especially the old tournaments. Putting this out, and it is a WIP. Need wording and formatting changes to this.

---

### CODE REVIEW

File: wiki/Tournaments/Country_Rankings/en.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+| 1 | ![][flag_JP] Japan | 6 | 1 | 2 |
+| 2 | ![][flag_TW] Taiwan | 2 | 4 |  |
+| 3 | ![][flag_HK] Hong Kong |  | 3 | 1 |
+| 4 | ![][flag_FR] France |  |  | 2 |
+| 5 | ![][flag_CA] Canada |  |  | 2 |
+| 5 | ![][flag_CN] China |  |  | 1 |
+
+#### Attendance
+
+| Country | 2011 | 2012 | \#3 | 2014 | 2015 | 2016 | 2017 | 2018 |
```
Comment (WalterToro) [inline wiki/Tournaments/Country_Rankings/en.md]: Would be better to replace (1st) (2nd) and (3rd) with the crowns?

Comment (Joehuu) [reply wiki/Tournaments/Country_Rankings/en.md]: Yeah, will apply that.

**Comment by TPGPL** [comment]
You should replace Countries that participated in osu! tournaments with Country Rankings in `Tournaments` and `Sitemap` pages

**Comment by Joehuu** [comment]
Stale, will remake another day.

**Pull request closed without merging**